cf-usb/cf-usb.yaml

```diff
@@ -136,7 +150,7 @@ jobs:
     - put: docker.mysql
       params:
         build: mysql-binary
-        load_base: docker.base-fixed
+        load_bases: [ docker.base-fixed, docker.buildbase-fixed ]
```

Comment:
This would probably look better in the other list format:
```yaml
load_bases:
- docker.base-fixed
- docker.buildbase-fixed
```

Reply:
I had a syntax error when I tried that at first.
Spaces vs tabs ?

Reply:
I believe tabs are banned in YAML, for Reasons.

Reply:
Heh. Will change.

Reply:
Done with commit 345074c906484e2478bf6d1289bd535795c298ff, in testing.